users-service/src/server.ls

```diff
@@ -130,3 +129,18 @@ function mongo-to-id entry
 function mongo-to-ids entries
   for entry in entries
     mongo-to-id entry
+
+
+function get-mongo-config
+  require('../service.yml').dependencies.mongo
+
+
+function get-mongo-address
+  mongo-config = get-mongo-config!
+  switch env
+    | \test => "mongodb://#{mongo-config.test}/space-tweet-users-test"
+    | \dev  => "mongodb://#{process.env.MONGO}/space-tweet-users-dev"
```

Comment:
In development, the Mongo container will be booted up by `exo-run`. In that case the users service needs the IP address of the Docker container running mongo. This information is conveyed in `exo-run` by setting the environment variable `MONGO`, it is not something user defined in `service.yml`.